_recipes/creamy-vegan-gnocchi-with-garlic-and-kale.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Creamy Vegan Gnocchi With Garlic and Kale
+category: Dinners
+tags: 
+  - hearty
+  - pasta
+  - vegan
+prepTime:
+  - PT5M
+cookTime:
+  - PT7M
+recipeYield: 3 meals
+recipeCuisine: Italian
+recipeIngredient:
+  - slug: olive oil
+    amount:
+      - 1 tbs
+  - slug: garlic
+    amount: 6 cloves
+  - slug: kale
+    amount: 1 bunch
+  - slug: cashews
+    amount:
+      - 75 g
+      - 1/2 cup
+  - slug: water
+    amount:
+      - 60 ml
+      - 1/2 cup
+  - slug: gnocchi
+    amount:
+      - 500 g
+  - slug: salt
+    amount:
+      - 1/4 tps
```

Comment:
I'd like to ensure every measure has a metric measure as the first. Volumetric measurements can be problematic in cooking, especially with salt which can be very course or very fine

```suggestion 
      - 4 g
      - 1/4 tsp
```